adafruit_portalbase/__init__.py

```diff
@@ -392,7 +392,7 @@ def _fill_text_labels(self, values):
         if self._text:
             value_index = 0  # In case values and text is not the same
             for i in range(len(self._text)):
-                if not self._text[i]["is_data"] or (value_index > len(values) - 1):
+                if (not self._text[i]["is_data"]) or (value_index > len(values) - 1):
```

Comment:
one more!
if (not self._text[i]["is_data"]) or (value_index > (len(values) - 1)):

Reply:
Oops, thanks